make_queue/templates/make_queue/course/registration_edit.html

```diff
@@ -81,9 +81,18 @@ <h1>
                 ;
             })
         ;
-        $(".clear-dropdown").click(function() {
+        $(".clear-dropdown").click(function () {
             $(this).closest("div.ui.field").find("div.dropdown").dropdown("clear");
         });
+
+        $("input[name='username']").focusout(function () {
+                $.ajax(langPrefix + "/reservation/json/" + $("input[name='username']").val() + "/", {
+                    success: function (data) {
+                        $("input[name='name']").val(data["full_name"]);
```

Comment:
Could we add a [warning message](https://semantic-ui.com/collections/message.html#warning) when the user cannot be found as well (In similar fashion to the one shown when adding a user)? With text like "The given username could not be found in NTNUs system. Are you sure you entered the correct username?". Would solve many of the problems we currently have with usernames sometimes being entered incorrectly.

Reply:
Added a large error message at the top of the page when the user is not found. The users should also be able to verify that the username entered matches with the full name field, so hope this becomes lesser of a problem in the future.